deploy/docker-ephemeral/build/Makefile

```diff
@@ -0,0 +1,216 @@
+# use DOCKER_ so we allow users to pass in values without conflicting with USERNAME, EMAIL, or somesuch already in their environments.
+DOCKER_USERNAME ?= wireserver
+DOCKER_REALNAME ?= Wire
+DOCKER_EMAIL    ?= backend@wire.com
```

Comment:
sorry for this commit on your branch, didn't mean to push this to your branch - but are you ok with these as default names?